docs/pages/machine-id/guides/hostcertificate.mdx

```diff
@@ -0,0 +1,403 @@
+
+First, create a basic configuration file using the following parameters:
+
+<Tabs>
+  <TabItem label="Token-based Joining">
+
+  <ScopedBlock scope={["oss", "enterprise"]}>
+
+  ```code
+  $ tbot configure \
+     --data-dir=/var/lib/teleport/bot \
+     --token=(=presets.tokens.first=) \
+     --join-method=token \
+     --certificate-ttl=1h0m0s \
+     --ca-pin=(=presets.ca_pin=) \
+     --auth-server=auth.example.com:3025
+  ```
+
+  </ScopedBlock>
+  <ScopedBlock scope={["cloud"]}>
+
+  ```code
+  $ tbot configure \
+     --data-dir=/var/lib/teleport/bot \
+     --token=(=presets.tokens.first=) \
+     --join-method=token \
+     --certificate-ttl=1h0m0s \
+     --ca-pin=(=presets.ca_pin=) \
+     --auth-server=example.teleport.sh:443
+  ```
+
+  </ScopedBlock>
+
+  </TabItem>
+  <TabItem label="IAM Method">
+
+  <ScopedBlock scope={["oss", "enterprise"]}>
+
+  ```code
+  $ tbot configure \
+     --data-dir=/var/lib/teleport/bot \
+     --token=iam-token \
+     --join-method=iam \
+     --certificate-ttl=1h0m0s \
+     --ca-pin=(=presets.ca_pin=) \
+     --auth-server=auth.example.com:3025
+  ```
+
+  </ScopedBlock>
+  <ScopedBlock scope={["cloud"]}>
+
+  ```code
+  $ tbot configure \
+     --data-dir=/var/lib/teleport/bot \
+     --token=iam-token \
+     --join-method=iam \
+     --certificate-ttl=1h0m0s \
+     --ca-pin=(=presets.ca_pin=) \
+     --auth-server=example.teleport.sh:443
+  ```
+
+  </ScopedBlock>
+
+  </TabItem>
+</Tabs>
+
+Replace the following fields with values from your own cluster.
+
+<ScopedBlock scope={["cloud"]}>
+
+- `token` is the token output by the `tctl bots add` command or the name of your IAM method token.
+- `ca-pin` is the CA Pin for your Teleport cluster, and is output by the `tctl bots add` command.
+- `data-dir` is where Machine ID writes its private data, including its own short-lived renewable certificates. These should not be used by applications and tools.
+- `auth-server` is the address of your Teleport Cloud Proxy Server, for example `example.teleport.sh:443`.
+- `certificate-ttl` is the option that will result in creation of a host certificate with Machine ID. The default TTL is for one hour, meaning that all generated certificates will expire after one hour's time.
+
+</ScopedBlock>
+<ScopedBlock scope={["oss","enterprise"]}>
+
+- `token` is the token output by the `tctl bots add` command or the name of your IAM method token.
+- `ca-pin` is the CA Pin for your Teleport cluster, and is output by the `tctl bots add` command.
+- `data-dir` is where Machine ID writes its private data, including its own short-lived renewable certificates. These should not be used by applications and tools.
+- `certificate-ttl` is the option that will result creation of a host certificate with Machine ID. The default TTL is for one hour, meaning that all generated certificates will expire after one hour's time.
+- `auth-server` is the address of your Teleport Auth Server, for example
```

Comment:
Teleport itself now supports `--proxy-server` or `--auth-server`. We should probably add the equivalent flags to tbot (cc @strideynet @timothyb89)

Reply:
Way ahead of ya https://github.com/gravitational/teleport/issues/16768 - I'll add some more details to that ticket though.